test/tools/libtesteth/Options.cpp

```diff
@@ -72,6 +74,26 @@ void printVersion()
 {
     cout << prepareVersionString() << "\n";
 }
+}
+
+void Options::setVerbosity(int _level)
+{
+    static std::ostringstream strCout;  // static string to redirect logs to
+    if (_level > verbosity)
+    {
+        verbosity = _level;
+        if (_level <= 0)
+        {
+            dev::setupLogging(-1);
+            std::cout.rdbuf(strCout.rdbuf());
+            std::cerr.rdbuf(strCout.rdbuf());
+        }
+        else if (_level == 1 || _level == 2)
+            dev::setupLogging(-1);
+        else
+            dev::setupLogging(_level);
```

Comment:
So there's no way for the user of testeth to set cpp logs verbosity to 0, 1 or 2, is it ok for you?

Reply:
usually the user of testeth is me ) 
by default testeth output logs for travis. it is like it has its own verbosity reserved for 0 1 and 2.